bin/zerobufCxx.py

```diff
@@ -832,23 +915,63 @@ def parse_members(self, fbsFile):
             is_byte_type = fbs_type == "byte" or fbs_type == "ubyte"
             value_type = ValueType(cxxtype, cxxtype_size, is_zerobuf_type, is_enum_type, is_byte_type)
 
+            properties[name] = {}
+            property = properties[name]
+
             if self.is_dynamic(attrib, fbsFile):
                 if len(attrib) == 2 and is_zerobuf_type:
                     member = DynamicZeroBufMember(name, value_type, dynamic_type_index)
+                    table = next(x for x in fbsFile.tables if x.name == cxxtype)
+                    properties[name] = table.json_schema
```

Comment:
It's python, so it's most probably a reference. I tried the same assignment as you described, and as it didn't work, I fell back to the code I put.